src/gl/uniform.c

```diff
@@ -110,21 +110,21 @@ void gl4es_glGetUniformfv(GLuint program, GLint location, GLfloat *params) {
     FLUSH_BEGINEND;
     CHECK_PROGRAM(void, program);
 
-    khint_t k;
+    khint_t k = 0;
```

Comment:
why the `= 0`? It seems like a useless affectation

Reply:
> 
> 
> why the `= 0`? It seems like a useless affectation

clang(12) diagnostic:
---cit---
.../src/gl/uniform.c(115,49) : warning: variable 'k' is uninitialized when used here [-Wuninitialized]
    k = kh_get(uniformlist, glprogram->uniform, k);
                                                ^
.../include/khash.h(472,45) : note: expanded from macro 'kh_get'
#define kh_get(name, h, k) kh_get_##name(h, k)
                                            ^
.../src/gl/uniform.c(113,14) : note: initialize the variable 'k' to silence this warning
    khint_t k;
             ^
              = 0
1 warning generated.
---end---

Reply:
Mmmm, ok, there is a (huge) mistake in this line, and that's why the warning trigger:
instead of `k = kh_get(uniformlist, glprogram->uniform, k);`
it should be `k = kh_get(uniformlist, glprogram->uniform, location);`
Not even sure how it can work like that.